tests/environments/minimal.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
```

Comment:
This would work:

```lua
local environments_folder_path = vim.fn.fnamemodify(debug.getinfo(1, "S").source:sub(2), ":p:h") .. "/"
local helpers = dofile(environments_folder_path .. "test-env-helpers.lua")

helpers.setup(".tests/minimal/site")
```